Dockerfile

```diff
@@ -15,6 +17,37 @@ RUN set -ex; \
     ; \
     rm -rf /var/lib/apt/lists/*
 
+# Build Python 2.7.9 from source
+RUN set -ex; \
+    curl -LO https://www.python.org/ftp/python/2.7.9/Python-2.7.9.tgz; \
+    tar -xzf Python-2.7.9.tgz; \
+    cd Python-2.7.9; \
+    ./configure --enable-shared; \
+    make; \
+    make install; \
+    cd ..; \
+    rm -rf /Python-2.7.9; \
+    rm Python-2.7.9.tgz
```

Comment:
It is unfortunate, but hopefully it's temporary - if pyOpenSSL can fix its import-related oddities, hopefully it'll work with PyInstaller and we can go back to using the package.

The problem with using a newer distro is that it upgrades glibc, which is bad for compatibility with older distros.